### DISCUSSION TITLE
App store price

### DISCUSSION THREAD

**Discussion by jchanal** [latenitefilms/BRAWToolbox#58] [General]
Hi, i would like to buy at mac app store but the price should be less than 47,99€  (36,39€)no ?
Maybe it's VAT but the price not corresponding.

I'm i wrong ?

Thanks.

Julien

**Comment by latenitefilms** [comment]
You can currently buy BRAW Toolbox for USD$79.99, however for the first week only as a thank you to our early adopters, it will be 50% off. ~~This discount ends 13th February 2023.~~

**EDIT:** _Apologies, the date above was incorrect. The discount is only valid for the first week, which means it ended on 5th of Feb, not 13th of Feb._

Looking at Apple's site this means it's currently:

USD 39.99
EUR 47.99
CNY 258.00
JPY 6,400
AUD 62.99

We don't provide Apple with exact pricing - we just provide it with a "tier" - currently it's Tier 40, and will increase to Tier 56.

**Reply by hendogfr** [reply]
Why did you change the offer ? It was 50% until February 13th originally.

I was on tour last week so I didn't have my laptop. Came back this morning Feb 6th, all pumped up and ready to buy your software and now the price doubled...Not cool.

**Reply by latenitefilms** [reply]
Apologies for any confusion! I made a mistake in that last post, as I correctly said "for the first week only", but accidentally wrote the wrong date. Sorry!

**Comment by jchanal** [comment]
I want buy to but 97€ !!!!!

**Reply by latenitefilms** [reply]
Unfortunately this is a very niche app with a relatively small group of users, so we need to charge a reasonable amount to justify the continued support and development. It's also something that Apple will "Sherlock" eventually, as I'm sure native BRAW support is on their roadmap. The original 50% discount was simply to say thank you to the early adopters who helped with beta testing and promoting BRAW Toolbox.

You can always use DaVinci Resolve for free if you want to convert BRAW to ProRes to edit in Final Cut Pro.

**Comment by latenitefilms** [comment]
@jchanal - Thanks to some changes to the Mac App Store, I have now been able to tweak the pricing model slightly.

BRAW Toolbox is now a one-time payment of **79.99**.

This is generally 79.99 in your local currency (i.e. **AUD79.99**, **USD79.99**, **CAD79.99**).

However, if your country doesn't have an equivalent of 79.99 (i.e. Rp1,299,000 in Indonesia), it will default to a **USD$79.99 equivalent**.

Hopefully this helps a bit!
